crates/rosty/tests/parameters.rs

```diff
@@ -28,6 +28,12 @@ fn test_parameter_api() {
         // And it should be changed
         assert_eq!(param.get::<i32>().await.unwrap(), 10);
 
+        // Change it to a boolean
```

Comment:
Why is this removed?

Reply:
It was broken, this needs to be fixed :)